### PR TITLE
Add valve.siml

### PR DESCRIPTION
**Pull request by Abrynos**

### CODE REVIEW

**Comment by Abrynos** [comment]
Am i doing this correctly?

**Comment by JustArchi** [comment]
Bonus points for mapping what user said in the response 😁

**Comment by Abrynos** [comment]
Does `<Match/>` work for dynamic sets?

**Comment by Thielak** [comment]
> Does `<Match/>` work for dynamic sets?

It does, but right now `<Match/>` would return the value of the first `$`  you put there

> Q> what cat wrong dog volvo servers?
> A> What is NOT wrong with cat?

To return the expected value you need to use `<Match At="n" />`, with n equal to the position of what you want to return.

I can't think of anything that would fit in place of the first wildcard `$` other than "is", so I'd probably change it to that to make the pattern a little clearer.
I should also note that `$` matches zero or more words, and I think in your example `*` or even better `_` (to give priority to your pattern and avoid conflicts with the default english siml base) would fit the bill better.

If I correctly understand what you were trying to accomplish (other than playing around of course, good job using sets ^^) I'd probably change things to this:
```xml		
<Model>
	<Pattern>what is wrong _ [volvo] (server|servers)</Pattern>
	<Response>What is NOT wrong <Match At="1" /> <Match At="2" /> <Match At="3" />?</Response>
</Model>
```
The 3 `Match` return the value of `_`, `[volvo]` and `(server|servers)` respectively, crafting an answer that more closely fits what the user told the bot:

> Q> what is wrong with volvo servers?
> A> What is NOT wrong with volvo servers?

> Q> what is wrong about valve servers?
> A> What is NOT wrong about valve servers?

Last thing to note for my example is that I placed a [non-breaking space](https://en.wikipedia.org/wiki/Non-breaking_space) (Alt+0160 on Windows) between each of the 3 Match tags, because otherwise the answer would look something like `What is NOT wrong withvolvoservers?`. Xml stuff, you could also use `xml:space="preserve"` but that's a story for another episode :p

File: Brain/ArchiBot/Custom/valve.siml [hunk 1]
```diff
@@ -2,13 +2,47 @@
...
+		<Model>
+			<Pattern>what is wrong with [volvo] servers</Pattern>
+			<Response>What is NOT wrong with <Match At="1"/> servers?</Response>
+		</Model>
+		<Model>
+			<Pattern>can you fix [volvo]</Pattern>
+			<Response>It's like trying to solve the halting problem :bite:</Response>
+		</Model>
+		<Model>
+			<Pattern>can you fix [volvo] servers</Pattern>
```
Comment (JustArchi) [inline Brain/ArchiBot/Custom/valve.siml]: You can declare multiple patterns and use one response instead of 2 gotos

**Pull request merged**